arviz/plots/tsplot.py

```diff
@@ -0,0 +1,430 @@
+"""Plot timeseries data."""
+import warnings
+import numpy as np
+
+from ..sel_utils import xarray_var_iter
+from ..rcparams import rcParams
+from .plot_utils import default_grid, get_plotting_function
+
+
+def plot_ts(
+    idata,
+    y,
+    x=None,
+    y_hat=None,
+    y_holdout=None,
+    y_forecasts=None,
+    x_holdout=None,
+    plot_dim=None,
+    holdout_dim=None,
+    num_samples=100,
+    backend=None,
+    backend_kwargs=None,
+    y_kwargs=None,
+    y_hat_plot_kwargs=None,
+    y_mean_plot_kwargs=None,
+    vline_kwargs=None,
+    textsize=None,
+    figsize=None,
+    legend=True,
+    grid=False,
+    axes=None,
+    show=None,
+):
+    """Plot timeseries and it's components.
+
+    Parameters
+    ----------
+    idata : InferenceData
+    y : str, variable name from observed_data
+    x : str, Optional
+        If none, coords of y dims
+    y_hat : str, from posterior_predictive, optional
+        Assumed to be of shape (chain, draw, *y.dims)
+    y_holdout : str, from observed_data, optional
+        Observed data after holdout
+    y_forecasts : str, from posterior_predictive, optional
+        Assumed shape (chain, draw, *)
```

Comment:
This is an example for my comment above. I'm not entirely certain what a holdout vs forecast is just reading the docstring

Reply:
Oh, that's an important observation! Thanks!